composer.json

```diff
@@ -36,6 +36,7 @@
         "psr/http-factory": "^1.0",
         "psr/http-message": "^1.0",
         "psr/log": "^1.0",
+        "sonata-project/admin-bundle": "^4.0@dev",
```

Comment:
Datagrid were a require-dev. Why now Admin is a require ?

Reply:
Because datagrid being a require-dev was a mistake, I got big errors when trying to use sonata-media 4 without having the datagrid-bundle

Reply:
Ref: https://github.com/Runroom/runroom-packages/pull/92/checks?check_run_id=2892702348